public/javascripts/application.js

```diff
@@ -455,7 +455,8 @@ $j(document).ready(function() {
         var msg = 'Sorry, we were unable to save your kudos';
         var data = $j.parseJSON(jqXHR.responseText);
 
-        if (data.errors && (data.errors.pseud_id || data.errors.ip_address)) {
+        // TODO: AO3-5859 Remove data.errors.pseud_id.
+        if (data.errors && (data.errors.pseud_id || data.errors.ip_address || data.errors.user_id)) {
```

Comment:
Does the check for `data.errors.pseud_id` have to be kept until [AO3-5859](https://otwarchive.atlassian.net/browse/AO3-5859)? I think this is the PR that changes the error keys.

Reply:
Yeah, I keep it because while deploying we'll have a mix of old and new versions.

Reply:
Isn't caching going to break the error messages during deploy regardless? People who've already opened the page before the deploy and press the kudos button after the deploy will get a nondescript "Sorry, we were unable to save your kudos" message instead of the specific "You have already left kudos here! :)" message.

Reply:
Good point, it still breaks in that case. I was thinking of the case where people load the JS file of the new version, but their kudos request gets routed to a server running the old version.

Reply:
I'm just not sure it's worth the effort of yet more incremental changes when it doesn't break anything important. It seems cleaner just to make the change and accept that some of the error messages will be less specific during the deploy, rather than adding yet more things to clean up later.

Reply:
Yeah, I can remove it.